RHEL/7/input/xccdf/services/ssh.xml

```diff
@@ -21,6 +21,47 @@ operator="equals" interactive="0">
 <value selector="120_minutes">7200</value>
 </Value>
 
+<Rule id="package_openssh-server_installed" severity="medium">
+<title>Install the OpenSSH Server Package</title>
+<description>
+The <tt>openssh-server</tt> package should be installed.
+<package-install-macro package="openssh-server"/>
+</description>
+<ocil clause="the package is not installed" >
+<package-check-macro package="openssh-server" />
+</ocil>
+<rationale>
+Without protection of the transmitted information, confidentiality, and
+integrity may be compromised because unprotected communications can be
+intercepted and either read or altered.
+</rationale>
+<ident cce="RHEL7-CCE-TBD" />
+<oval id="package_openssh-server_installed" />
+<ref nist="SC-8" disa="2418,2420,2421,2422"
+ossrg="SRG-OS-000423-GPOS-00187,SRG-OS-0004230GPOS-00188,SRG-OS-000423-GPOS-00189,SRG-OS000423-GPOS-00190"
+stigid="040260" />
+</Rule>
+
+<Rule id="service_sshd_enabled" severity="medium">
+<title>Enable the SSH Server</title>
```

Comment:
For consistency with package_openssh-server_installed, can we change this to "Enable the OpenSSH Service"